examples/guide/Cargo.toml

```diff
@@ -6,8 +6,13 @@ name = "guide"
 publish = false
 version = "0.10.0"
 
+[features]
+default = ["train"]
+train = ["burn/train"]
+train-minimal = ["burn/train-minimal"]
+
 [dependencies]
-burn = {path = "../../burn", features = ["train", "wgpu"]}
```

Comment:
I think the guide needs the `train` feature flag!

Reply:
It's included by the default features above. If you'd prefer not to have the extra complexity in the guide, can you suggest a simple other model this example could depend on? I based the example off of your custom-training-loop one.